### PR TITLE
Deleted project visible bug

### PR DESCRIPTION
**Pull request by abinashpa**
## Summary
https://www.loom.com/share/6f20757600f5423dbe1b89e880f46a38
## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to
      not work as expected)
- [ ] This change requires a documentation update

### Checklist:

- [x] I have manually tested all workflows
- [x] I have performed a self-review of my own code
- [ ] I have added automated tests for my code

### CODE REVIEW

**Comment by supriya3105** [comment]
@abinashpa The CI Checks are failing for this .

**Review by supriya3105 — APPROVED** [review]

**Comment by supriya3105** [comment]
@apoorv1316 Please test this @sudeeptarlekar Please review .

**Review by apoorv1316 — APPROVED** [review]

File: spec/models/client_spec.rb [hunk 1]
```diff
@@ -130,13 +132,15 @@
...
-          result = [project_1, project_2].map do | project |
+          results = [project_1, project_2].map do | project |
             {
               id: project.id, name: project.name, team: project.project_member_full_names,
               minutes_spent: project.timesheet_entries.where(work_date: from..to).sum(:duration)
             }
           end
-          expect(client.project_details(time_frame)).to eq(result)
+          results.map do |result|
+            expect(client.project_details(time_frame)).to include(result)
```
Comment (sudeeptarlekar) [inline spec/models/client_spec.rb:142]: Check if we can use `match_array` from rspec here.

**Review by sudeeptarlekar — CHANGES_REQUESTED** [review]

**Comment by supriya3105** [comment]
@abinashpa Lets work on the review comments on priority and close this PR.

**Review by sudeeptarlekar — APPROVED** [review]
LGTM

**Pull request merged**